CIP-high-assurance/README.md

```diff
@@ -0,0 +1,173 @@
+---
+CIP: XXX
+Title: High Assurance Best Practices
+Category: Meta
+Status: Draft
+Authors:
+ - Romain Soulat <romain.soulat@iohk.io>
+Implementers: []
+Discussions:
+ - Intersect's Plutus Working Group
```

Comment:
```suggestion
 - https://github.com/cardano-foundation/CIPs/pull/926
```
The "Working Group" item will need to be converted into a URL if & when ready with such a link; otherwise it will have to be removed.  You can include a YAML label like `Intersect-Plutus-Working-Group` if needed to disambiguate from the usual contents of this list which are GitHub PR URLs.